src/constants/eula.ts

```diff
@@ -0,0 +1 @@
+export const EULA_URL = "https://covid-safe-paths-eula.herokuapp.com"
```

Comment:
Could this be a configuration item? In other words, will this be something that could potentially change from HA to HA or even from one country build to the other? Could we benefit from having the constant as part of the environment variables? If we don't and it is a static value, we might also move this constant to the `Eula` component since it is only used there and this file seems too specific in a too general folder. Thoughts?